src/lib/NovaCellAPIClient.ts

```diff
@@ -116,4 +122,14 @@ export class NovaCellAPIClient {
   readonly libraryProgram = this.withCellId(LibraryProgramApi)
   readonly libraryRecipeMetadata = this.withCellId(LibraryRecipeMetadataApi)
   readonly libraryRecipe = this.withCellId(LibraryRecipeApi)
+
+  readonly storeObject = this.withCellId(StoreObjectApi)
+  readonly storeCollisionComponents = this.withCellId(
+    StoreCollisionComponentsApi,
+  )
+  readonly storeCollisionScenes = this.withCellId(StoreCollisionScenesApi)
+
+  readonly cell = this.withCellId(CellApi)
+  readonly application = this.withCellId(ApplicationApi)
+  readonly system = this.withCellId(SystemApi)
```

Comment:
cell, application and system also work on a by cell basis?

Reply:
OH good catch